FlowCryptTests/ExtensionTests.swift

```diff
@@ -69,13 +69,15 @@ extension ExtensionTests {
 extension ExtensionTests {
     func test_calendar_date_formatting() {
         // 18:34
-        let sameDayDate = Date(timeIntervalSince1970: 1622561652)
+        let sameDayDate = Date()
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "HH:mm"
         // Jan 24
         let sameYearDate = Date(timeIntervalSince1970: 1611506052)
         // Jan 24, 2020
         let otherYearDate = Date(timeIntervalSince1970: 1579883652)
         
-        XCTAssertEqual(DateFormatter().formatDate(sameDayDate), "18:34")
+        XCTAssertTrue(dateFormatter.date(from: DateFormatter().formatDate(sameDayDate)) != nil)
```

Comment:
This is not a robust way to test it - should at least test for a particular pattern like `/^\d\d:\d\d$/`